server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -721,6 +722,21 @@ thruToken
    : (THROUGH | THRU)
    ;
 
+sqlDataTypesCaluse
+   : TYPE IS (sqlDataTypesCaluseForm1 | sqlDataTypesCaluseForm2)
+   ;
+
+sqlDataTypesCaluseForm1
+   : sqlDataTypes LPARENCHAR integerLiteral RPARENCHAR
+   ;
+
+sqlDataTypesCaluseForm2
+   : RESULT_SET_LOCATOR VARYING
+   ;
+
+sqlDataTypes
```

Comment:
There are way more than these two types. Please, remove it and focus on `RESULT_SET_LOCATOR` in this PR.